server/v2/stf/core_event_service.go

```diff
@@ -53,34 +59,3 @@ func (em *eventManager) EmitKV(eventType string, attrs ...event.Attribute) error
 	em.executionContext.events = append(em.executionContext.events, event.NewEvent(eventType, attrs...))
```

Comment:
We could also fill in `Event.Data` here too lazily converts attribute based events to JSON